docs/connector-development/connector-builder-ui/overview.md

```diff
@@ -1,57 +1,48 @@
 # Connector Builder Intro
 
-The connector builder UI provides an intuitive UI on top of the [low-code YAML format](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/yaml-overview) and uses built connectors for syncs within the same workspace directly from within the UI. We recommend using it to iterate on your low-code connectors.
+Connector Builder is a no-code tool that’s part of the Airbyte UI. It provides an intuitive user interface on top of the [low-code YAML format](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/yaml-overview) and lets you develop a connector to use in data syncs without ever needing to leave your Airbyte workspace. The connector builder offers the most straightforward method for building and maintaining connectors.
 
-:::note
-The connector builder UI is in beta, which means it’s still in active development and may include backward-incompatible changes. Share feedback and requests with us on our Slack channel or email us at feedback@airbyte.io
+We recommend that you determine whether the connector you want can be built with the connector builder before looking at the Low-Code CDK or Python CDK. 
```

Comment:
Do we still have documentation to help determine if the connector can be built with Connector Builder? If yes, we should link that here

Reply:
adding!